### PR TITLE
docs: update ordering on the sidebar

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -13,9 +13,9 @@
   - [Proofreading a Curriculum File](how-to-proofread-files.md)
   - [How to Translate the Website](how-to-translate-the-website.md)
 - **Optional Guides**
-  - [Catch outgoing emails locally](how-to-catch-outgoing-emails-locally.md)
-  - [Set up freeCodeCamp on WSL](how-to-setup-wsl.md)
+  - [Set up freeCodeCamp on Windows(WSL)](how-to-setup-wsl.md)
```
Comment (nhcarrigan) [inline docs/_sidebar.md:16]:
```suggestion
  - [Set up freeCodeCamp on Windows Subsystems for Linux](how-to-setup-wsl.md)
```

Just a thought, but I wonder if we should spell this out for translation purposes?

Comment (raisedadead) [reply docs/_sidebar.md:16]: I am concerned if this fits on the sidebar. It's already spelt out in the guide in the title and elsewhere.

<details>
<summary>Screenshot</summary>

image


</details>

Comment (nhcarrigan) [reply docs/_sidebar.md:16]: Ah, true - a good point.

Comment (raisedadead) [reply docs/_sidebar.md:16]: Good to resolve and review then?

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Pull request merged**